### PR TITLE
[FEATURE] Revised project structure completely

### PR DESCRIPTION
**Pull request by Zethson**
Time to professionalize things :)

Installation: `make install`

Running: `SEIR --help`

### CODE REVIEW

**Comment by ghost** [comment]
Why do we need the makefile? The package can already be automatically installed with pip

**Comment by Zethson** [comment]
"Why do we need the makefile? The package can already be automatically installed with pip"

And it can still be installed via pip.
It's just a wrapper.

pip install . works
python setup.py install works

make install just wraps python setup.py clean --all install

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+[*.bat]
+indent_style = tab
+end_of_line = crlf
+
+[LICENSE]
+insert_final_newline = false
+
+[Makefile]
+indent_style = tab
+
```
Comment (ghost) [inline .editorconfig:22]: What does 'editorconfig' add on top of PEP8?

File: AUTHORS.rst [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+Development Lead
+----------------
+
+* Lukas Heumos <lukas.heumos@posteo.net>
+
+Contributors
+------------
+
+None yet. Why not be the first?
```
Comment (ghost) [inline AUTHORS.rst]: I'd just suggest a list of contributors. No need to identify a Dev Lead.

Comment (Zethson) [reply AUTHORS.rst]: Absolutely. Sorry, this is autogenerated. Will fix

File: .editorconfig (hunk 1 above)
Comment (Zethson) [reply .editorconfig:22]: Streamlines how some editors treat spaces, tabs etc. 
Some editors use by default tabs, but spaces are required.

Usually only important in edge cases, but doesn't hurt.

PEP8 is tested via flake8 (see Github workflow)

**Comment by Zethson** [comment]
@solanpaa 

So flake8 fails with:

```
./SEIR/seir.py:175:9: E129 visually indented line with same indent as next logical line
./SEIR/seir.py:247:54: W605 invalid escape sequence '\m'
./SEIR/seir.py:263:25: E127 continuation line over-indented for visual indent
./SEIR/seir.py:338:9: E125 continuation line with same indent as next logical line
./SEIR/seir.py:376:13: E741 ambiguous variable name 'I'
./SEIR/seir.py:392:17: E741 ambiguous variable name 'I'
./SEIR/seir.py:396:17: E741 ambiguous variable name 'I'
./SEIR/seir.py:508:9: F841 local variable 'ICUwindow' is assigned to but never used
```

If you are fine with this PR I would fix the merge conflicts and then you can take care of PEP8 conformity later?

**Comment by ghost** [comment]
I think the new structure is good, let's go with this. @Zethson if you can fix the conflicts and then we can merge this to master.

**Comment by Zethson** [comment]
@solanpaa Great. Will do.

Just to reinforce it: I don't want to take sole credit or something here. If you find any spots where only my name is mentioned, please inform me and I will change this to Biohackathon Biostats Team or something along this lines.

It's all autogenerated (by a tool that will be released in the coming months :) )

**Comment by ghost** [comment]
It's all good 😉 There are also some remnants from my original codebase which refer to me, we should replace also those.

**Pull request merged**